lib/.eslintrc.yaml

```
@@ -37,6 +37,8 @@ rules:
       message: "Use `const { Atomics } = globalThis;` instead of the global."
     - name: Buffer
       message: "Use `const { Buffer } = require('buffer');` instead of the global."
+    - name: DOMException
+      message: "Use lazy function `const { lazyDOMException } = require('internal/util');` instead of the global class."
```

nit: to keep the consistency with the other messages
```suggestion
      message: "Use lazy function `const { lazyDOMException } = require('internal/util');` instead of the global."
```